priv/repo/seeds/tools.exs

```diff
@@ -53,21 +55,22 @@ defmodule Parzival.Repo.Seeds.Tools do
 
   def seed_announcements do
     # admins = Repo.all(where(User, role: :admin))
+    admins = Repo.all(User)
```

Comment:
Shouldn't be it only admins?

Reply:
As of now, we do not have roles in the users schema. 

In a future PR with that change, I will update this code. :pray: